cos-fleetshard-operator/src/main/java/org/bf2/cos/fleetshard/operator/connector/ConnectorController.java

```diff
@@ -305,7 +305,6 @@ private UpdateControl<ManagedConnector> measure(
             .tags(tags)
             .tag("cos.connector.id", connector.getSpec().getConnectorId())
             .tag("cos.deployment.id", connector.getSpec().getDeploymentId())
```

Comment:
We probably can also remove the deployment_id, we dont use it, @shivamG640 to confirm

Reply:
@lgarciaaco @shivamG640 do we actually use cos.connector.id and cos.deployment.id ?

Reply:
deploymentId you can delete but @shivamG640 should confirm that dashboards dont use it. Connector id we need

Reply:
For which reason we use it ?
Note that this is the operator, no the sync

Reply:
Sorry for missing the old conversation!
@lgarciaaco, I could only find the usage of these metrics on this [file](https://github.com/bf2fc6cc711aee1a0c2a/cos-observability-resources/blob/main/resources/grafana/overview.yaml#L502)
@lburgazzoli, This metric is related to reconciliations and we don't monitor reconciliation per connectors so we can remove the connector_id and deployment_id.